### PR TITLE
CA-374877: Double check the iperf test result against the tcpdump result

### PR DESCRIPTION
**Pull request by stephenchengCloud**
To avoid accidently failure of iperf test

### CODE REVIEW

File: plugins/autocertkit [hunk 1]
```diff
@@ -401,6 +403,101 @@ def run_ack_logrotate(session, args):
...
+        mip = validate_exists(args, 'mip')
+        username = validate_exists(args, 'username')
+        password = validate_exists(args, 'password')
+        # Only setup tcpdump if not control domain
+        # - that already has tcpdump installed.
+        i = 0
+        limit = 10
+        while i < limit:
+            try:
+                install_tcpdump(session, vm_ref, mip, username, password)
```
Comment (DeliZhangX) [inline plugins/autocertkit:437]: You can call `install_rpm()`

File: plugins/autocertkit [hunk 2]
```diff
@@ -401,6 +403,101 @@ def run_ack_logrotate(session, args):
...
+    iface = validate_exists(args, 'iface')
+
+    kill_processes('tcpdump')
+
+    # Check for tcpdump existence
+    if not process_running(TCPDUMP):
+        log.debug("tcpdump is not running - start process")
+
+        # Start tcpdump - need to ensure we don't block
+        call = "nohup /usr/sbin/tcpdump -s 128 -n -i %s src host %s -w tcpdump.pcap &" % (iface, ip)
```
Comment (DeliZhangX) [inline plugins/autocertkit:467]: define `/usr/sbin/tcpdump` at file beginning

File: plugins/autocertkit [hunk 3]
```diff
@@ -401,6 +403,101 @@ def run_ack_logrotate(session, args):
...
+        log.debug("About to start tcpdump server (%s)" % call)
+        subprocess.Popen(call, shell=True)
```
Comment (DeliZhangX) [inline plugins/autocertkit:469]: call `make_local_call()`

Comment (stephenchengCloud) [reply plugins/autocertkit:469]: make_local_call waits for the process return which would block the main process.

File: plugins/autocertkit [hunk 4]
```diff
@@ -401,6 +403,89 @@ def run_ack_logrotate(session, args):
...
+    iface = validate_exists(args, 'iface')
+
+    kill_processes('tcpdump')
+
+    # Check for tcpdump existence
+    if not process_running(TCPDUMP):
+        log.debug("tcpdump is not running - start process")
+
+        # Start tcpdump - need to ensure we don't block
+        call = "nohup %s -s 128 -n -i %s src host %s -w tcpdump.pcap &" % (TCPDUMP, iface, ip)
```
Comment (DeliZhangX) [inline plugins/autocertkit:455]:
```
tcpdump -i eth0 -s 128 -n -l | awk  '{ if ($(NF-1) == "length") {sum+=$NF; printf("%d %d\n",$NF, sum);}}'
```

**Review by DeliZhangX — APPROVED** [review]

**Review by xihuan-citrix — APPROVED** [review]

**Pull request merged**